Sources/ApolloCodegenLib/Frontend/CompilationResult.swift

```diff
@@ -44,6 +44,10 @@ public class CompilationResult: JavaScriptObject {
       "\(name) on \(rootType.debugDescription)"
     }
 
+    public func hash(into hasher: inout Hasher) {
+      hasher.combine(name)
```

Comment:
There can never be 2 operations with the same name, as graphql-js would fail validation. So this is safe